main.go

```diff
@@ -295,13 +295,27 @@ func checkExpectations(expectations []Expectation, scriptValues map[string]strin
 			expected := expectation.Arguments[0]
 			actual := scriptValues[expectation.Arguments[1]]
 			if expected != actual {
-				errStr := fmt.Sprintf("Expected %s to be equalt to %s but it's clearly not!!!", expected, actual)
+				errStr := fmt.Sprintf("Expected %s to be equal to %s but it's clearly not!!!", expected, actual)
 				if expectation.Fatal {
 					fmt.Printf("FATAL: %s\n-----\n", errStr)
 					os.Exit(0)
 				}
 				return errors.New(errStr)
 			}
+		case "string_contains":
+			expected := expectation.Arguments[0]
+			searchable_string := scriptValues[expectation.Arguments[1]]
+			if !strings.Contains(searchable_string, expected) {
+				errStr := fmt.Sprintf("Expected - %s\n to contain '%s' but it clearly does not!!!", searchable_string, expected)
+				// NOTE: this will raise on fatal
```

Comment:
YaI clearly want to bubble up the failures  that are not fatal and publish them at the end of the run with the stage they happened in.  I just have not had the time yet - maybe you can write an issue about it remove this comment and tackle it next?  :P

Reply:
Hahaha excellent, issue created!